charts/portworx/v2.8.1/templates/_helpers.tpl

```diff
@@ -95,6 +95,35 @@ release: {{ .Release.Name | quote }}
 {{- end -}}
 {{- end -}}
 
+
+{{- define "px.getCSIProvisionerImage" -}}
+{{- if semverCompare "<=1.6-0" .Capabilities.KubeVersion.GitVersion -}}
+    {{ "docker.io/openstorage/csi-provisioner:v1.6.1-1" }}
+{{- else if semverCompare "<=1.9-0" .Capabilities.KubeVersion.GitVersion -}}
+    {{ "docker.io/openstorage/csi-provisioner:v2.2.2-1" }}
+{{- else -}}
+    {{ "docker.io/openstorage/csi-provisioner:v3.0.0-1" }}
+{{- end -}}
+{{- end -}}
+
+{{- define "px.getCSISnapshotterImage" -}}
+{{- if semverCompare "<=1.9-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
this should be for 1.16.x and earlier:
https://github.com/portworx/px-installer/blob/master/px-spec-websvc/csi.go#L270-L271

Reply:
good catch